lib/client/identityfile/identity.go

```diff
@@ -334,7 +334,7 @@ func Write(cfg WriteConfig) (filesWritten []string, err error) {
 			ClusterAddr:         cfg.KubeProxyAddr,
 			Credentials:         cfg.Key,
 			TLSServerName:       cfg.KubeTLSServerName,
-		}); err != nil {
+		}, false); err != nil {
```

Comment:
Why is this always `false`? Should not this take "load all CAs" setting into account?